### PR TITLE
tools/deps.sh fixes

### PR DESCRIPTION
**Pull request by amada95**

### CODE REVIEW

File: tools/deps.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ for pkg in "${packages[@]}"; do
 	if [[ "$VERBOSE" == "true" ]]; then
 		install_package $pkg
 	else
-		install_package $pkg &>/dev/null
+		install_package $pkg &>/dev/null
```
Comment (Andy-Python-Programmer) [inline tools/deps.sh:47]: oops the ` ` :trollface:

File: tools/deps/deps_darwin [hunk 2]
```diff
@@ -47,4 +47,4 @@ packages=(
 )
 
 function install_package() { brew install $1; }
-function query_package() { brew list | grep -q $1; }
+function query_package() { brew list | { grep -q $1; cat > /dev/null;} }
```
Comment (Andy-Python-Programmer) [inline tools/deps/deps_darwin:50]: `cat > /dev/null` to resolve `SIGPIPE` would overwrite the exit status from grep. So, store the exit status of grep and return that.

Comment (amada95) [reply tools/deps/deps_darwin]: This should be fixed now!

File: tools/deps.sh [hunk 3]
```diff
@@ -46,12 +46,12 @@ for pkg in "${packages[@]}"; do
 	else
 		install_package $pkg &>/dev/null
 	fi
-
+
```
Comment (Andy-Python-Programmer) [inline tools/deps.sh:49]: nit: more space :trollface:

**Review by Andy-Python-Programmer — APPROVED** [review]
LGTM!

**Comment by Andy-Python-Programmer** [comment]
Thanks @amada95!

**Pull request merged**